BUILD

```diff
@@ -217,21 +216,28 @@ docker_container_image(
     visibility = ["//test:__subpackages__"],
 )
 
-docker_container_bundle(
-    name = "assemble-docker-bundle",
-    images = {
-        "{}/{}/{}:{{DOCKER_VERSION}}".format(
-            deployment_docker['docker.release'], deployment_docker['docker.organisation'], deployment_docker['docker.repository']
-        ): ":assemble-docker",
-        "{}/{}/{}:latest".format(
-            deployment_docker['docker.release'], deployment_docker['docker.organisation'], deployment_docker['docker.repository']
-        ): ":assemble-docker",
-    }
+docker_container_push(
+    name = "deploy-docker-latest",
+    image = ":assemble-docker",
+    format = "Docker",
+    registry = deployment_docker["docker.release"],
+    repository = "{}/{}".format(
+        deployment_docker["docker.organisation"],
+        deployment_docker["docker.repository"],
+    ),
+    tag = "latest"
 )
 
-docker_push(
-    name = "deploy-docker",
-    bundle = ":assemble-docker-bundle",
+docker_container_push(
+    name = "deploy-docker-versioned",
+    image = ":assemble-docker",
+    format = "Docker",
+    registry = deployment_docker["docker.release"],
+    repository = "{}/{}".format(
+        deployment_docker["docker.organisation"],
+        deployment_docker["docker.repository"],
+    ),
+    tag_file = ":VERSION",
```

Comment:
this target reads the tag name from VERSION file. The `deploy-docker-latest` target has a fixed tag called `label`.